spec/term/1.0.md

```diff
@@ -0,0 +1,204 @@
+<!-- draft -->
+# `vt6/term1.0` - Universal terminal capabilities and input/output handling
+
+The canonical URL for this document is <https://vt6.io/std/term/1.0/>.
+
+**This is a non-normative draft.**
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+This document uses the predefined property types from [section 2.4 of `vt6/core1.0`](https://vt6.io/std/core/1.0/#section-2-4).
+
+## 1. Scope
+
+This module defines how a terminal works, how clients' standard output is reflected on the terminal, and how user input is presented on clients' standard input.
+Besides [`vt6/core`](https://vt6.io/std/core/), this is the only module that we expect to be available in **every** VT6 server implementation across the entire feature spectrum, from graphical terminal emulators all the way down to continuous stationery printers.
```

Comment:
Except muxers and proxies...

Reply:
Muxers and proxies still need to implement `term`. They *are not* the terminal, but they have an internal model of a terminal that they present towards their own clients. A muxer/proxy that only has `core` is less than useless.

Reply:
So proxies and muxers would just forward all term-related messages from the terminal to the client and not modify it. Makes sense.

Reply:
Well, not "not modify it". The only modification they do is multiplexing it onto a single connection.

Reply:
That's what I meant.